_includes/footer.html

```diff
@@ -5,53 +5,82 @@
   </div>
   {% endunless %}
 
-  <div class="usa-footer__secondary-section bg-primary">
-    <div class="grid-container">
-      <div class="display-flex flex-align-center margin-bottom-4">
-        <a href="https://www.whitehouse.gov/omb/"
-          ><img
-            class="width-10 display-block"
-            src="{{ '/assets/img/eop-omb.png' | relative_url }}"
-            alt="Office of Management and Budget"
-        /></a>
-        <div class="separator display-flex flex-align-center">
-          <a href="https://digital.gov"
-            ><img
-              class="height-3 display-block"
-              src="{{ '/assets/img/digital-gov-logo--white.png' | relative_url }}"
-              alt="Digital.gov"
-          /></a>
+
+
+  <div class="usa-identifier">
```

Comment:
Can we move this to its own include since it's a separate component?

Reply:
Moved to _includes/identifier.html